### ISSUE TITLE
训练 icdar 2015  一直卡在dataloader

### ISSUE THREAD

**Issue by yuanyuan1993** [WenmuZhou/PSENet.pytorch#83]
log信息：
make: Entering directory '/home/yuanyuan/Desktop/chinese_ocr/PSENet.pytorch/pse'
make: 'pse.so' is up to date.
make: Leaving directory '/home/yuanyuan/Desktop/chinese_ocr/PSENet.pytorch/pse'
2020-06-09 16:43:07 INFO      utils.py: logger init finished
2020-06-09 16:43:07 INFO      train.py: {'Lambda': 0.7,
 'OHEM_ratio': 3,
 'backbone': 'resnet152',
 'checkpoint': '',
 'data_shape': 640,
 'display_input_images': False,
 'display_interval': 10,
 'display_output_images': False,
 'end_lr': 1e-07,
 'epochs': 600,
 'gpu_id': '0',
 'lr': 0.0001,
 'lr_decay_step': [200, 400],
 'lr_gamma': 0.1,
 'm': 0.5,
 'n': 6,
 'output_dir': 'output/psenet_icd2015_resnet152_4gpu_author_crop_adam_MultiStepLR_authorloss',
 'pretrained': True,
 'restart_training': True,
 'scale': 1,
 'seed': 2,
 'show_images_interval': 50,
 'start_epoch': 0,
 'testroot': 'data2/dataset/ICD15/test',
 'train_batch_size': 4,
 'trainroot': 'data2/dataset/ICD15/train',
 'warm_up_epoch': 6,
 'warm_up_lr': 1e-05,
 'weight_decay': 0.0005,
 'workers': 12}
2020-06-09 16:43:07 INFO      train.py: train with gpu 0 and pytorch 1.1.0
2020-06-09 16:43:09 INFO      resnet.py: load pretrained models from imagenet
2020-06-09 16:43:11 INFO      train.py: train dataset has 1000 samples,250 in dataloader

一直卡在这上面，请问一下是什么情况

**Comment by WenmuZhou** [comment]
num_workers 设为0看看

**Comment by yuanyuan1993** [comment]
> num_workers 设为0看看

感谢！
